### PR TITLE
[doc] list templateflow templates that need to be prefetched for fmriprep i…

### LINKED ISSUE #2195: List the templates required by fmriprep in the faq
In the faq, we could list the templates that are required by the pipeline, that need to be prefetched in the absence of internet connection on cluster node. Unless I haven't seen it in the doc, it is pretty much a trial and error, you have to wait for the pipeline to crash with a missing template to know. 

It seems that the pipeline requires: `'MNI152NLin2009cAsym', 'MNI152NLin6Asym', 'OASIS30ANTs'`
Can anyone confirm?

### PR DESCRIPTION
**Pull request by bpinsard**
…nternal usage

## Changes proposed in this pull request

Improve documentation to fix #2195 
Some templateflow templates are required by fmriprep independent of requested output-spaces.
In the case were it needs to be prefetched, these need to be prefetched as well.

## Documentation that should be reviewed

faq.rst

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -192,6 +192,9 @@ compute nodes are isolated, or in PC/laptop enviroments if you are travelling.
 query for a template resource that has not been previously accessed.
 If you know what are the templates you are planning to use, you could
 prefetch them using the Python client.
+In addition to the output-spaces that you specify, fmriprep will use
+'MNI152NLin2009cAsym', 'MNI152NLin6Asym' and 'OASIS30ANTs' templates, that
```
Comment (mgxd) [inline docs/faq.rst:196]: I'm fairly sure ``MNI152NLin6Asym`` is only required if aroma or cifti outputs are specified. 
```suggestion
In addition to the output-spaces that you specify, *fMRIPrep* will use
``MNI152NLin2009cAsym``, ``MNI152NLin6Asym`` and ``OASIS30ANTs`` templates, that
```

**Comment by mgxd** [comment]
this looks good, could you update the styling as mentioned in https://github.com/poldracklab/fmriprep/pull/2196#discussion_r443548159?

File: docs/faq.rst [hunk 2]
```diff
@@ -192,6 +192,10 @@ compute nodes are isolated, or in PC/laptop enviroments if you are travelling.
 query for a template resource that has not been previously accessed.
 If you know what are the templates you are planning to use, you could
 prefetch them using the Python client.
+In addition to the output-spaces that you specify, *fMRIPrep* will use
```
Comment (oesteban) [inline docs/faq.rst:195]:
```suggestion
In addition to the ``--output-spaces`` that you specify, *fMRIPrep* will
```

File: docs/faq.rst [hunk 3]
```diff
@@ -192,6 +192,10 @@ compute nodes are isolated, or in PC/laptop enviroments if you are travelling.
...
+``MNI152NLin2009cAsym`` and ``OASIS30ANTs`` templates,
```
Comment (oesteban) [inline docs/faq.rst:196]:
```suggestion
internally require the ``MNI152NLin2009cAsym`` template.
If the ``--skull-strip-template`` option is not set, then ``OASIS30ANTs``
will be used.
```

File: docs/faq.rst [hunk 4]
```diff
@@ -192,6 +192,10 @@ compute nodes are isolated, or in PC/laptop enviroments if you are travelling.
...
+(as well as ``MNI152NLin6Asym`` if ``--cifti-output`` or ``--use-aroma`` is specified),
```
Comment (oesteban) [inline docs/faq.rst:197]:
```suggestion
Finally, either ``--cifti-output`` or ``--use-aroma`` arguments require ``MNI152NLin6Asym``,
```

File: docs/faq.rst [hunk 5]
```diff
@@ -187,11 +187,17 @@ How do you use TemplateFlow in the absence of access to the Internet?
...
 *TemplateFlow* will require Internet access the first time it receives a
 query for a template resource that has not been previously accessed.
 If you know what are the templates you are planning to use, you could
 prefetch them using the Python client.
+In addition to the ``--output-spaces`` that you specify, *fMRIPrep* will
+internally require the ``MNI152NLin2009cAsym`` template.
+If the ``--skull-strip-template`` option is not set, then ``OASIS30ANTs``
+will be used.
+Finally, either ``--cifti-output`` or ``--use-aroma`` arguments require ``MNI152NLin6Asym``,
+that also need to be prefetched.
```
Comment (oesteban) [inline docs/faq.rst:200]:
```suggestion
that also needs to be prefetched.
```

Comment (mgxd) [inline docs/faq.rst:200]:
```suggestion
Finally, both the ``--cifti-output`` and ``--use-aroma`` arguments require ``MNI152NLin6Asym``.
```

**Pull request merged**